src/apprentice.c

```diff
@@ -0,0 +1,2380 @@
+#include "global.h"
+#include "apprentice.h"
+#include "battle_tower.h"
+#include "data2.h"
+#include "event_data.h"
+#include "event_object_movement.h"
+#include "field_player_avatar.h"
+#include "international_string_util.h"
+#include "item.h"
+#include "item_menu.h"
+#include "main.h"
+#include "malloc.h"
+#include "menu.h"
+#include "new_game.h"
+#include "party_menu.h"
+#include "random.h"
+#include "script.h"
+#include "script_menu.h"
+#include "sound.h"
+#include "string_util.h"
+#include "strings.h"
+#include "task.h"
+#include "text.h"
+#include "constants/apprentice.h"
+#include "constants/items.h"
+#include "constants/songs.h"
+#include "constants/species.h"
+#include "constants/moves.h"
+
+#define PLAYER_APPRENTICE gSaveBlock2Ptr->playerApprentice
```

Comment:
Was this define really necessary?

Reply:
It gets a bit hard to read with field_B8[field_B1]